aten/src/ATen/native/LinearAlgebraUtils.h

```diff
@@ -30,6 +32,46 @@ static char to_blas(TransposeType trans) {
   TORCH_INTERNAL_ASSERT(false, "Invalid transpose type");
 }
 
+template<class Vec>
+static inline Vec contiguous_strides_template(const IntArrayRef sizes, const bool f_contig=false) {
+  static_assert(std::is_same<IntArrayRef::value_type, typename Vec::value_type>::value,
+                "Incompatible integral type of sizes and strides");
+  // f_contig chooses between Fortran (F-contiguous) and C-contiguous strides
```

Comment:
Unfortunately, this comment is not correct.
`f_contig` chooses between C-contiguous strides (`f_contig=false`) and C-contiguous strides for N-2 dimensions + F-contiguous strides for the last 2 dimensions (`f_contig=true`).
This function is applicable only to batches of matrices and I think it should be reflected in the function name.

```python
In [1]: import numpy as np
# note that NumPy returns strides using bytes
In [2]: np.ones((2, 3, 4, 5, 6), order='C').strides
Out[2]: (2880, 960, 240, 48, 8)

In [3]: np.ones((2, 3, 4, 5, 6), order='F').strides
Out[3]: (8, 16, 48, 192, 960)
```

Reply:
That's a fair point. Corrected.